client/ayon_harmony/plugins/publish/validate_scene_settings.py

```diff
@@ -155,11 +156,10 @@ def process(self, instance):
         ):
             msg = (
                 "Handles included in calculation. Remove handles in DB"
-                " or extend frame range in timeline."
+                " or extend frame range in timeline.\n"
             )
```

Comment:
Why not just log this as a separate message?


```suggestion
            self.log.info(
                "Handles included in calculation. Remove handles in DB"
                " or extend frame range in timeline."
            )
```

Reply:
log info and above will show in the publish report, and should also be present in the validation report. So `log.info`, `log.warning` and `log.error` should all be fine for actionable items for the user.

Only `log.debug` appears on Details tab only AFAIK.